peekingduck/pipeline/nodes/model/yolo_face.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Face Detection model
```

Comment:
Let's make this consistent with mtcnn (Fast face detection model that works best with unmasked faces)
```suggestion
Fast face detection model that can distinguish between masked and unmasked faces
```